packages/dx-scheduler-core/src/plugins/month-view/computeds.js

```diff
@@ -1,10 +1,36 @@
 import moment from 'moment';
+import {
+  sortAppointments,
+  viewPredicate,
+  toPercentage,
+} from './../../utils';
+import {
+  findOverlappedAppointments,
+  adjustAppointments,
+  unwrapGroups,
+} from './../week-view/helpers';
```

Comment:
Can we move this stuff to the `common` folder? 🤔 We use it for both week and  month views.